.github/workflows/ci.yml

```diff
@@ -91,7 +99,15 @@ jobs:
             ${{ runner.OS }}-
       - run: npm ci --legacy-peer-deps
       - run: echo CLOUDMERSIVE_KEY=${{secrets.CLOUDMERSIVE_KEY}} >> .env
+      - name: Configure Datadog Test Visibility
+        uses: datadog/test-visibility-github-action@v1
+        with:
+          languages: js
+          service: gogovsg
+          api_key: ${{ secrets.DD_API_KEY }}
       - run: npm run test:e2e-headless
+        env:
+          NODE_OPTIONS: -r ${{ env.DD_TRACE_PACKAGE }} --import ${{ env.DD_TRACE_ESM_IMPORT }}
```

Comment:
unsure if dd is compatible with testcafe; might be why it's breaking. 

can try without the `--import` flag first, otherwise can remove totally